lib/index.js

```diff
@@ -164,7 +164,7 @@ function morphAttrs(fromNode, toNode) {
         if (attr.specified !== false) {
             attrName = attr.name;
             if (!foundAttrs.hasOwnProperty(attrName)) {
-                fromNode.removeAttributeNS(attr.namespaceURI, attr.name);
+                fromNode.removeAttributeNode(attr);
```

Comment:
Attributes should also check namespaces.
eg. attrName can be same for 
xlink:href="" href=""

And then you can use removeAttributeNS
https://developer.mozilla.org/en-US/docs/Web/API/Element/removeAttributeNS